lib/galaxy/tools/parameters/wrapped_json.py

```diff
@@ -88,7 +92,7 @@ def _data_input_to_path(v):
     elif input_type == "boolean":
         json_value = _cast_if_not_none(value_wrapper, bool)
     elif input_type == "select":
-        if input.multiple:
+        if input.multiple and profile >= '20.05':
```

Comment:
`profile` is a float, so you'd have to use `if input.multiple and profile >= 20.05:` 
But we usually use `packaging.version.parse(profile) >= packaging.version.parse('20.05')`

Reply:
Thanks for the suggestion.